internal/hcsoci/hcsdoc_lcow.go

```diff
@@ -50,12 +50,11 @@ func createLCOWSpec(coi *createOptionsInternal) (*specs.Spec, error) {
 }
 
 func setWindowsNetworkNamespace(coi *createOptionsInternal, spec *specs.Spec) {
+	spec.Windows = &specs.Windows{}
```

Comment:
[this](https://github.com/microsoft/hcsshim/blob/6fef88c5a9870e32f776421af44a51f6c5ee8ad1/internal/hcsoci/hcsdoc_lcow.go#L32) is where the actual problem happens.